stdio needed for sscanf(), stdlib is included by smi.h but let's include it
anyway for exit().
--- a/libsmi.m4
+++ b/libsmi.m4
@@ -12,7 +12,10 @@ AC_DEFUN([AM_PATH_LIBSMI],
   AC_MSG_CHECKING([whether to enable libsmi])
   AC_TRY_RUN([ /* libsmi available check */
 #include <smi.h>
-main()
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+int main(void)
 {
   int current, revision, age, n;
   const int required = 2;
